### ISSUE TITLE
RStudio Server menus difficult to use with Safari/VoiceOver on macOS

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5119]
### System details

    RStudio Edition : Server (Ubuntu 18)
    RStudio Version : 1.3.320
    OS Version      : macOS 10.14.5
    R Version       : 3.4.4

### Steps to reproduce the problem
- Load RStudio Server in Safari on macOS
- Start VoiceOver
- VO+Tab (to ignore next keypress)
- Ctrl+Option+F (to send focus to File menu)
- Navigate menus with Up/Down/Left/Right arrow keys

### Describe the problem in detail
- VoiceOver is often out-of-sync with what is currently selected (i.e. where the keyboard focus is located)
- Sometimes it eventually catches up, other times it never reads what is currently selected
- Often it will re-read the same menu item multiple times (seems like it is reading the currently selected item again as it loses selection)
- Other times it seems like it is reading every single menu item, even if you've skipped over them and hit Ctrl to try and shut it up

### Describe the behavior you expected
- Should be easy to navigate the menus with VoiceOver and be confident that what is says you are on is what you are on
- Behavior is much better with NVDA on Firefox, and JAWS on Chrome (both on Windows)
- Behavior is somewhat better with VoiceOver on Chrome on macOS; not perfect, but better (contradicts the general guidance that Safari is the reference standard for testing VoiceOver...)
